examples/pghostname_zig/src/main.zig

```diff
@@ -0,0 +1,28 @@
+const std = @import("std");
+const pgzx = @import("pgzx");
+
+comptime {
+    pgzx.PG_MODULE_MAGIC();
+
+    pgzx.PG_FUNCTION_V1("pghostname_zig", pghostname_zig);
```

Comment:
Hm... I haven't tried this, but I wonder if `pgzx.PG_EXPORT(@This())` will do the trick (this will require you to declare your function as `pub`).

Reply:
This worked ✅